build/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+# See LICENSE.txt for license information.
+ARG DOCKER_BUILD_IMAGE=golang:1.15
+ARG DOCKER_BASE_IMAGE=ubuntu:20.04
```

Comment:
There was, yes, I wanted to use a utility that wasn't available in Alpine, but there is no longer any good reason because I went a different way with implementation. There are a couple of things that are wrong in this Dockerfile now, so I'll adjust before merging.

Reply:
That said I wouldn't be surprised if we need to make some changes to this Dockerfile as we're preparing to deploy this service